TaskWithMaxMinSortForArray/TaskWithMaxMinSortForArray.cpp

```diff
@@ -0,0 +1,61 @@
+﻿#include <iostream>
+
+int searchMax(int* array, size_t len) {
+	int max = array[0];
+	for (int i = 1; i < len; i++)
+	{
+		if (array[i] > max)
+		{
+			max = array[i];
+		}
+	}
+	return max;
+}
+
+int searchMin(int* array, size_t len) {
+	int min = array[0];
+	for (int i = 1; i < len; i++)
+	{
+		if (array[i] < min)
+		{
+			min = array[i];
+		}
+	}
+	return min;
+}
+
+void bubbleSort(int* array, size_t len) {
+	for (int i = 0; i < len - 1; i++) {
+		for (int j = 0; j < len - i - 1; j++) {
+			if (array[j] > array[j + 1]) {
+				int temp = array[j];
+				array[j] = array[j + 1];
+				array[j + 1] = temp;
+			}
+		}
+	}
+}
+
+int main(int argc, char* argv[])
+{
+	if (argc == 1)
+	{
+		std::cout << "No command line arguments " << std::endl;
+		return 1;
+	}
+	const int sizeOfArray = argc - 1;
+	int* array = new int[sizeOfArray];
+	for (size_t i = 1; i < sizeOfArray + 1; i++)
```

Comment:
The condition feels more natural for me if it's i< argc instead of i < sizeOfArray + 1. Because the first operation is to take a value from argv, then convert to string, at the end assign to the buffer of the sizeOfArray size. In my mind I, and possibly all developers, link the iteration with what I take data from. But you can merge with it. No error.

Reply:
Fixed